docs/book/guides/common-usecases/feature-store.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I guess this needs to be fleshed out, right?

Reply:
Yes I wrote in my PR description that I'm still working on it. I'll add this this morning.